spec/Section 6 -- Execution.md

```diff
@@ -799,39 +808,45 @@ MergeSelectionSets(fields):
   - Append all selections in {fieldSelectionSet} to {selectionSet}.
 - Return {selectionSet}.
 
-### Handling Field Errors
+<a name="sec-Handling-Field-Errors">
+  <!-- This link exists for legacy hyperlink support -->
+</a>
+
+### Handling Execution Errors
 
-A _field error_ is an error raised from a particular field during value
+An _execution error_ is an error raised from a particular field during value
 resolution or coercion. While these errors should be reported in the response,
 they are "handled" by producing a partial response.
 
 Note: This is distinct from a _request error_ which results in a response with
 no data.
 
-If a field error is raised while resolving a field, it is handled as though the
-field returned {null}, and the error must be added to the {"errors"} list in the
-response.
+If an execution error is raised while resolving a field (either directly or
```

Comment:
Should we do underscores around `execution error` everywhere?

Reply:
The general pattern is to do it at least once per headed section, but not to worry about it after that. I generally do it when I feel it adds clarity.